features/admin_merge.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Merge Articles
+  As a blog administrator
+  In order to organize my blog entries
+  I want to be able to merge articles in my blog
+
+  Background:
+    Given the blog is set up
+    And each article has a comment
+    And I am logged into the admin panel
+    Given I am on the first article edit page
+
+  Scenario: Admins can see merge option
```

Comment:
Should there be a scenario where a non-admin should not be able to see the merge option?